schemas/1.0/pruned_application.json

```diff
@@ -5,37 +5,113 @@
   "description": "Attributes of a pruned criminal legal aid application",
   "type": "object",
   "properties": {
-    "id": { "type": "string" },
-    "parent_id": { "type": ["string", "null"] },
-    "schema_version": { "type": "number" },
-    "reference": { "type": "integer" },
-    "application_type": { "type": "string", "enum": ["initial", "post_submission_evidence", "change_in_financial_circumstances"] },
-    "created_at": { "type": "string", "format": "date-time" },
-    "submitted_at": { "type": "string", "format": "date-time" },
-    "date_stamp": { "type": ["string", "null"], "format": "date-time" },
-    "returned_at": { "type": ["string", "null"], "format": "date-time" },
-    "reviewed_at": { "type": ["string", "null"], "format": "date-time" },
-    "status": { "type": "string", "enum": ["submitted", "returned", "superseded"] },
+    "id": {
+      "type": "string"
+    },
+    "parent_id": {
+      "type": [
+        "string",
+        "null"
+      ]
+    },
+    "schema_version": {
+      "type": "number"
+    },
+    "reference": {
+      "type": "integer"
+    },
+    "application_type": {
+      "type": "string",
+      "enum": [
+        "initial",
+        "post_submission_evidence",
+        "change_in_financial_circumstances"
+      ]
+    },
+    "created_at": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "submitted_at": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "date_stamp": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "format": "date-time"
+    },
+    "returned_at": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "format": "date-time"
+    },
+    "review_status": {
+      "type": "string",
+      "enum": [
+        "application_received",
+        "returned_to_provider",
+        "ready_for_assessment",
+        "assessment_completed"
+      ],
+      "readOnly": true
+    },
```

Comment:
Added attribute ☝️  for review

Changes below here in this file are just formatting 👇